src/test/java/twitterapp/src/TwitterAppResourceTest.java

```diff
@@ -166,10 +195,10 @@ public void testBadFilter() throws Exception{
         assertEquals(Response.Status.INTERNAL_SERVER_ERROR, Response.Status.fromStatusCode(r.getStatus()));
     }
     @Test
-    public void throwExceptionWhenTimelineDoesNotPrint() throws Exception{
+    public void throwExceptionWhenHomeTimelineDoesNotPrint() throws Exception{
```

Comment:
maybe rename this to be more concise? you can explain the logic of the test in comments. also space.